tests/rptest/tests/rpk_config_test.py

```diff
@@ -146,24 +175,31 @@ def test_config_set_json(self):
         node = self.redpanda.get_node(n)
         rpk = RpkRemoteTool(self.redpanda, node)
         key = 'rpk'
-        value = '{"tune_aio_events":true,"tune_cpu":true,"tune_disk_irq":true}'
+        value = '{"tuneAioEvents":true,"tuneCpu":true,"tuneDiskIrq":true}'
```

Comment:
TODO: Define support for camelCase, snake_case, or both naming conventions for properties.